stable/fairwinds-insights/values.yaml

```diff
@@ -496,28 +506,39 @@ postgresql:
     certificatesSecret: fwinsights-postgresql-ca
     certFilename: tls.crt
     certKeyFilename: tls.key
-  postgresqlHost: insights-postgresql
+  # -- Host for postgresql (CloudNativePG cluster name)
+  postgresqlHost: insights-postgres-rw
+  port: 5432
+  # -- Storage configuration for the PostgreSQL cluster
+  storage:
+    size: 10Gi
+    storageClass: standard
+  # -- Authentication configuration
   auth:
     username: postgres
     database: fairwinds_insights
     existingSecret: fwinsights-postgresql
     secretKeys:
       adminPasswordKey: postgresql-password
-  primary:
-    service:
-      # -- Port of the Postgres Database
-      port: 5432
-    persistence:
-      # -- Create Persistent Volume with Postgres
-      enabled: true
-    # -- Resources section for Postgres
-    resources:
-      limits:
-        cpu: 1
-        memory: 1Gi
-      requests:
-        cpu: 75m
-        memory: 256Mi
+  # -- PostgreSQL configuration parameters
+  parameters:
```

Comment:
before we had resource limits and requires
```
    resources:
      limits:
        cpu: 1
        memory: 1Gi
      requests:
        cpu: 75m
        memory: 256Mi
```
We should probably have the same for the new chart, then using the limits of 1cpu and 1GB mem, which via PGTune is
```
max_connections = 50
shared_buffers = 256MB
effective_cache_size = 768MB
maintenance_work_mem = 64MB
checkpoint_completion_target = 0.9
wal_buffers = 7864kB
default_statistics_target = 100
random_page_cost = 1.1
effective_io_concurrency = 200
work_mem = 4519kB
huge_pages = off
min_wal_size = 1GB
max_wal_size = 4GB
```  
Most of it matches so I think it makes sense, so we just have to **set resource and limits**

Reply:
done